.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+src/database
```

Comment:
I think the `dist` folder generated by babel should be here too

Reply:
Done. Please resolve the conversation.